client/components/Editor/schemas/citation.js

```diff
@@ -24,9 +27,9 @@ export default {
 			},
 		],
 		toDOM: (node) => {
-			const { href, id, count } = node.attrs;
-
-			const { citationsRef, citationInlineStyle } = node.type.spec.defaultOptions;
+			const { href, id, count, customLabel } = node.attrs;
+			const { citationsRef } = node.type.spec.defaultOptions;
+			const { citationInlineStyle } = node.type.spec.defaultOptions;
```

Comment:
Is there a reason for these two destructures from `node.type.spec.defaultOptions` being separated?

Reply:
No, that was pretty silly.